configure

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+if [ -z "$PYTHON_BIN_PATH" ]; then
+  PYTHON_BIN_PATH=$(which python3 || which python || true)
+fi
+CONFIGURE_DIR=$(dirname "$0")
+
+function ask_question() {
+  if [[ -z "$1" ]]; then
+    echo "\`ask_question\` received no argument"
+    exit 1
+  fi
+
+  while :
+  do
+    read -r -p "${1} " response
+
+    case "$response" in
+      [yY])
+          echo "1"
+          break
+          ;;
+      [nN])
+          echo "0"
+          break
+          ;;
+      *)
+          echo "Incorrect input received: \`${response}\`. Expected: 'Y' or 'N'"
+          ;;
+    esac
+  done
+}
+
+# ====================== VSCode Development Container ======================= #
```

Comment:
In the interest of keeping the top level directory as thin as possible, I'd want the PR that adds this new file to also move [`configure_bazel.py`](https://github.com/openxla/iree/blob/main/configure_bazel.py) to [`build_tools/bazel/`](https://github.com/openxla/iree/tree/main/build_tools/bazel), and then call that from this script.

Reply:
Agreed

Reply:
Done